### PR TITLE
Feature/simulation api

### PR DESCRIPTION
**Pull request by steersbob**

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -0,0 +1,2 @@
+VUE_APP_API_URI=http://localhost:9000
+VUE_APP_HISTORY_URI=${VUE_APP_API_URI}/history
```
Comment (Gaya) [inline .env.development]: Nice, didn't know this was possible

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+
+  eventbus:
+    image: rabbitmq:alpine
+
+  influx:
+    image: influxdb:alpine
+    # volumes:
```
Comment (Gaya) [inline docker-compose.yml]: Need to leave these in?

File: vue.config.js [hunk 3]
```diff
@@ -1,13 +1,5 @@
-const apiMocker = require('connect-api-mocker');
-
 module.exports = {
   lintOnSave: true,
-  devServer: {
```
Comment (Gaya) [inline vue.config.js]: Do we need to remove the mock data in favour of using the docker container? I thought it was a nice option to mock the api like this. Maybe we should turn it into a setting in the `.env` files.

File: src/core/fetch.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { serialize, deserialize } from '@/core/units/parseObject';
 
-const host = '/api';
+const host = process.env.VUE_APP_API_URI;
```
Comment (Gaya) [inline src/core/fetch.ts]: I feel like adding `/datastore` here makes more sense than in the api calls in the other places of the source.

Makes it less redundant and less of a pain if `datastore` were to change at some point.

File: docker-compose.yml (hunk 2 above)
Comment (steersbob) [reply docker-compose.yml]: If you uncomment it, you get persistent data storage. That seemed like behavior that's occasionally useful for development, but not often enough to add it as a configuration option.

File: vue.config.js (hunk 3 above)
Comment (steersbob) [reply vue.config.js]: We can, but then we'd have to make the mocks match the actual API.

Probably best to discuss various implementations on thursday.

File: src/core/fetch.ts (hunk 4 above)
Comment (steersbob) [reply src/core/fetch.ts]: For `fetch.ts`, host refers to the entire backend - not just the datastore.

Probably need some semi-dynamic config somewhere. "datastore" and "history" are very stable, but each controller also has its own subdomain.

**Pull request merged**